### ISSUE TITLE
Field: Parent/Child - Enhancement

### ISSUE THREAD

**Issue by kiranetph** [Serhioromano/cobalt-9#164]
After reading the latest Blogpost I remembered a feature request for the Parent/Child field.
In an early version of Mighty Resources, the field was able to display several levels of relations.
It was later changed to display only 1 more level.
Maybe the old functionality can be restored under an "Advanced Option" to avoid an endless loop, but still give the option to show more than 1 Level.
Thanks for listening ... 

Still waiting for the next release for testing the improved features.

**Comment by KEZEO** [comment]
Yes would be great, I loved mighty ressources for such features.
